### DISCUSSION TITLE
Got error when starting edex-ui on mac

### DISCUSSION THREAD

**Discussion by nasuf** [GitSquared/edex-ui#984] [Questions] [answered]
![image](https://user-images.githubusercontent.com/13514662/102945566-108aa800-44f9-11eb-9a58-0c00e229c1c5.png)
macOS Catalina 10.15.7

**Comment by wrac4242** [comment]
make sure your internet is connected, as it is trying to connect to an invalid ip address

**Comment by GitSquared** [comment, marked as answer]
Hi! This looks like a software issue. We track those in the Issues tab, and this specific problem is tracked here: #982
